src/test/javascript/spec/component/tutorial-groups/course-tutorial-groups/course-tutorial-group-detail.component.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CourseTutorialGroupDetailComponent } from 'app/overview/tutorial-group-details/course-tutorial-group-detail/course-tutorial-group-detail.component';
+import { LoadingIndicatorContainerStubComponent } from '../../../helpers/stubs/loading-indicator-container-stub.component';
+import { TutorialGroupDetailStubComponent } from '../stubs/tutorial-group-detail-stub.component';
+import { TutorialGroupsService } from 'app/course/tutorial-groups/services/tutorial-groups.service';
+import { MockProvider } from 'ng-mocks';
+import { AlertService } from 'app/core/util/alert.service';
+import { Router } from '@angular/router';
+import { MockRouter } from '../../../helpers/mocks/mock-router';
+import { simpleOneLayerActivatedRouteProvider } from '../../../helpers/mocks/activated-route/simple-activated-route.providers';
+import { generateExampleTutorialGroup } from '../helpers/tutorialGroupExampleModels';
+import { HttpResponse } from '@angular/common/http';
+import { TutorialGroup } from 'app/entities/tutorial-group/tutorial-group.model';
+import { of } from 'rxjs';
+describe('CourseTutorialGroupDetailComponent', () => {
+    let fixture: ComponentFixture<CourseTutorialGroupDetailComponent>;
+    let component: CourseTutorialGroupDetailComponent;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [CourseTutorialGroupDetailComponent, TutorialGroupDetailStubComponent, LoadingIndicatorContainerStubComponent],
+            providers: [
+                MockProvider(TutorialGroupsService),
+                MockProvider(AlertService),
+                { provide: Router, useClass: MockRouter },
+                simpleOneLayerActivatedRouteProvider(
```

Comment:
Do you know about `convertToParamMap()`?

Reply:
Good idea! Removed the simple implementation in 404a61f9e625e0e40fb810485d800baaea28b685